lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -168,11 +168,11 @@ defmodule ExUnit.DocTest do
 
   Options can also be given:
 
-    * `:except` - generate tests for all functions except those listed
-      (list of `{function, arity}` tuples, and/or `:moduledoc`).
+    * `:except` - generates tests for all functions except those listed
+      (list of `{function, arity}` tuples, and/or `:moduledoc`)
 
-    * `:only`   - generate tests only for functions listed
-      (list of `{function, arity}` tuples, and/or `:moduledoc`).
```

Comment:
Okay, now I get it, and I think what we've been doing:

* comes down to
* this. This is a sentence

It's not perfect but I prefer this to inconsistent trailing periods (in the same bullet list ofc!).